docs/how-to-guides/authentication-with-dex-gangway.md

```diff
@@ -111,11 +111,6 @@ variable "gangway_session_key" {
 
 # Dex component configuration.
 component "dex" {
-  # NOTE: This name should match with the contour component configuration
-  # `ingress_hosts`
-  ingress_host = "dex.<CLUSTER_NAME>.<DOMAIN.NAME>"
-
-  issuer_host = "https://dex.<CLUSTER_NAME>.<DOMAIN_NAME>"
```

Comment:
Do we want to remove `issuer_host` as well?

Reply:
No, I don' this this should be removed, neither should `ingress_host`